configs/ner_sm.cfg

```diff
@@ -0,0 +1,137 @@
+[paths]
+vectors = null
+init_tok2vec = null
+
+[system]
+gpu_allocator = null
+seed = 0
+
+[nlp]
+lang = "en"
+pipeline = ["tok2vec","tagger","parser","ner"]
+tokenizer = {"@tokenizers":"spacy.Tokenizer.v1"}
+disabled = []
+before_creation = null
+after_pipeline_creation = null
+
+[nlp.after_creation]
+@callbacks = "replace_tokenizer"
+
+[components]
+
+[components.ner]
+factory = "ner"
+moves = null
+update_with_oracle_cut_size = 100
+
+[components.ner.model]
+@architectures = "spacy.TransitionBasedParser.v1"
+state_type = "ner"
+extra_state_tokens = false
+hidden_width = 64
+maxout_pieces = 2
+use_upper = true
+nO = null
+
+[components.ner.model.tok2vec]
+@architectures = "spacy.Tok2Vec.v1"
+
+[components.ner.model.tok2vec.embed]
+@architectures = "spacy.MultiHashEmbed.v1"
+width = 256
+attrs = ["NORM","PREFIX","SUFFIX","SHAPE"]
+rows = [5000,2500,2500,2500]
+include_static_vectors = true
+
+[components.ner.model.tok2vec.encode]
+@architectures = "spacy.MaxoutWindowEncoder.v1"
+width = 256
+depth = 8
+window_size = 1
+maxout_pieces = 3
+
+[components.parser]
+source = "output/en_core_sci_sm_parser_tagger/model-best"

```

Comment:
oof, this is pretty hard to tie back to the project.yml file. Is there a way to pass these components in via the `spacy train` cli instead?

Reply:
Yeah, looks like overrides can be passed in https://nightly.spacy.io/api/cli#train. seems better than this, i agree